proxy/core/src/test/scala/io/cloudstate/proxy/crdt/AbstractCrdtEntitySpec.scala

```diff
@@ -28,6 +28,8 @@ object AbstractCrdtEntitySpec {
       |akka.actor.provider = cluster
       |# Make the tests run faster
       |akka.cluster.distributed-data.notify-subscribers-interval = 50ms
+      |akka.actor.allow-java-serialization = on
```

Comment:
@jroper Hmm, this doesn't seem desirable—are we using Java Serialization?

Reply:
@sean-walsh Pinging Sean as well

Reply:
Well, this is in `test` configration, so using Java serialization could be tolerated if necessary. But is it really necessary?